utils/__init__.py

```diff
@@ -47,3 +47,17 @@ def generate_invite_url(discord_bot_application_id: int, discord_guild_id: int)
         scopes=("bot", "applications.commands"),
         disable_guild_select=True,
     )
+
+def is_user_inducted(guild: discord.Guild, user: discord.Member) -> bool:
```

Comment:
I suggest this function is moved to `utils/tex_bot.py:TeXBot` as a method. This will allow the use of `self.main_guild` without needing to pass in the guild from the calling context.